server/hap2/hatohol/hap2_nagios_ndoutils.py

```diff
@@ -117,6 +118,11 @@ def __parse_url(self, url):
 
         return server, port, database
 
+    def __convert_to_nagios_time(self, hatohol_time):
+        over_second = hatohol_time.split(".")[0]
+        nag_datetime = datetime.datetime.strptime(over_second, '%Y%m%d%H%M%S')
```

Comment:
Please avoid unnecessary variable abbreviation If you can.
It makes hard to understand what variables mean.